compiler/luci/pass/include/luci/Pass/FuseMulWithDivPass.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_FUSE_MUL_WITH_DIV_PASS_H__
+#define __LUCI_FUSE_MUL_WITH_DIV_PASS_H__
+
+#include <loco.h>
+
+#include <luci/ModulePass.h>
+
+namespace luci
+{
+
+/**
+ * @brief Pass to fuse mul(one of the input is const scalar) and div(one of the input is const
+ * scalar) as div
+ *
+ * For detailed subgraph pattern to be fused, please check its implementation.
+ */
```

Comment:
I'm not sure of these comment... it would be better to check other Pass header files and follow like them.

Reply:
Done